mpsl/doc/cx.rst

```diff
@@ -0,0 +1,73 @@
+.. _mpsl_cx:
+
+Radio coexistence feature
+#########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The radio coexistence feature allows the application to interface with several types of packet traffic arbiters (PTAs).
+This allows for better radio performance for devices using multiple interfering radios simultaneously, like a combination of |BLE| and Wi-Fi, or IEEE 802.15.4 and Wi-Fi.
+The PTA arbitrates the requested radio operations between all radios to avoid interference, increasing the performance of the radio protocols enabled simultaneously.
+The exact arbitration algorithm is dependent on the specific PTA used.
+
+.. note::
+  The radio coexistence feature is not needed for the arbitration between |BLE| and IEEE 802.15.4 in dynamic multiprotocol applications, as the dynamic multiprotocol does not use PTA for the arbitration.
+
+Limitations
+***********
+
+This experimental feature has several limitations:
+
+* It supports only the IEEE 802.15.4 protocol.
+* It supports only the nRF52 SoC family.
+* It does not use radio operation priorities, as the radio protocol requests each operation with a priority value of ``1``.
+* The PTA can deny only TX operations.
+  Denying RX or idle listening has no effect.
+* Aborting a TX operation takes a few dozens of microseconds.
+
+Implementation
+**************
+
+To use the radio coexistence feature, you can either select the implementation available in NCS or replace it with another implementation compatible with your PTA.
+Each implementation exposes the same API to allow for an easier replacement that does not require modifications to the radio protocols.
+
+The radio coexistence implementation provided in NCS is the following:
+
+* *Thread Radio Coexistence*.
+  This PTA interface is compatible with the 3-wire PTA interface defined in the *Thread Radio Coexistence recommendations* document.
```

Comment:
Thread Radio Coexistence recommendations should be link, or at the very least point to where to find the document in question.